File: strings/notes.py
# Zoey Sosa, Strings notes python

# strings is a data type that holds any info surround by quotation marks "" ' '

#note = "vienna's class"

#name = input("what is ypur first name\n").strip().capitalize()

#print(f"hello {name} welcome to my program!")
#print("this is your name "+  name)

#print(note)

sentence = "the quick brown fox jumps over the lazy dog"

#print(len(sentence))
#print(sentence[4])
start = sentence.find("brown")
length = len("brown fox")
print(sentence[start:start+length])

File: strings/notes.c
//zoey sosa, Strings notes C
#include <stdio.h>
#include <string.h>

//char subject[50];
//char name[]= "Victoria";
//char sentence[] = 

int main(void){
    //printf("What class are you in?\n");
    //scanf("%s", subject);
    //fgets(subject, sizeof(subject), stdin);
    //printf("you are in %s, that is a cool class", subject);
    //name[0] = 'T';
    //name[1] = 'o';
    //name[2] = 'r';
    //name[3] = 'i';
    //printf("%s", name);
    //printf("%lu\n", sizeof(sentence));
    //printf("%d\n",strlen(sentence));
    char one[]= "hello";
    char two[]= "world";
    char three[]= "welcome to my program";
    printf("%s\n",one);
    strcat(one, two);
    printf("%s\n", two);
    strcat(three,one); //can only concattenate 2 things at a time
    return 0;
}
sgv jsngoenoengwegmgpowme